crypto/property/property.c

```diff
@@ -362,6 +362,41 @@ int ossl_method_store_remove(OSSL_METHOD_STORE *store, int nid,
     return 0;
 }
 
+static void alg_do_one(IMPLEMENTATION *impl,
+                       void (*fn)(void *method, void *fnarg), void *fnarg)
+{
+    fn(impl->method.method, fnarg);
+}
+
+struct alg_do_each_data_st {
+    void (*fn)(void *method, void *fnarg);
+    void *fnarg;
+};
+
+static void alg_do_each(unsigned long idx, ALGORITHM *alg, void *arg)
```

Comment:
Function signature is 
void (ossl_uintmax_t, void *, void *)

Reply:
The cross compile builds caught something our usual CIs missed!

Reply:
Which is great - it saves it appearing on some obscure third party platform.

Reply:
Fixed